runtime/crab/src/lib.rs

```diff
@@ -676,7 +676,10 @@ impl OnRuntimeUpgrade for CustomOnRuntimeUpgrade {
 		migration::move_pallet(b"Instance0DarwiniaBalances", b"Balances");
 		migration::move_pallet(b"Instance1DarwiniaBalances", b"Kton");
 
-		migration::move_pallet(b"Instance0Collective", b"Instance2Collective");
+		// Tech.Comm to Instance2
+		migration::move_pallet(b"Instance1Collective", b"Instance2Collective");
+		// Council to Instance1
+		migration::move_pallet(b"Instance0Collective", b"Instance1Collective");
```

Comment:
This does not the correspond changes in Runtime:

https://github.com/darwinia-network/darwinia/compare/v0.10.1...xavier-mmr-patch#diff-c23cc93db5b2f8c1b19170f6ce738217642392f312028b87011d240406b255b3R258

Reply:
Check out this commit -> [prefer not to break the order of the instances](https://github.com/darwinia-network/darwinia/pull/675/commits/0bb448e7fce735df58ba84fbcbe2fd1716a3ab05).

There are just a few keys in collective to be migrated, so I decide to keep the order.

Reply:
Yes，but please take a look at my link and the order in runtime was changed and need to updated together.

Reply:
The storage prefix concept is different with the pallet index, it is using InstanceX generic to derive the prefix?

Reply:
Do you mean the index in `construct_runtime`. That's call index, not the pallet index.